src/app/devices/components/DeviceHeaderForms/DeviceHeaderForms.test.tsx

```diff
@@ -20,6 +20,7 @@ describe("DeviceHeaderForms", () => {
           <DeviceHeaderForms
             headerContent={{ view: DeviceHeaderViews.ADD_DEVICE }}
             setHeaderContent={jest.fn()}
+            devices={[]}
```

Comment:
The rest of the changes were lint or type issues exposed when I was looking at the the react-scripts upgrade.